sdk/storage/inc/datalake/datalake_utilities.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+#include <map>
+
+#include "common/crypt.hpp"
+
+namespace Azure { namespace Storage { namespace DataLake {
+
+  std::map<std::string, std::string> ParseDataLakePropertiesString(const std::string& dataLakePropertiesString)
```

Comment:
These two functions should be moved to protocol layer

Reply:
Swagger knows nothing about how this should be parsed or serialized. I don't think this should be put in protocol layer.

Reply:
okay, fine